### ISSUE TITLE
crayon-animate has gone from npm!!! 

### ISSUE THREAD

**Issue by acim** [alshdavid/crayon#64]

**Comment by d0x2f** [comment]
crayon-svelte is gone too, @alshdavid ?

**Comment by acim** [comment]
Yes, some of the packages depend on some deprecated libraries so they are gone. It's a pity, this is great router but I have to abandon it.

**Comment by 90minutes** [comment]
This is a really good router, I already use it in a production project and I couldn't replace it easily, so I republished the packages under a different name:
crayon-vue => crayon-router-vue
crayon-react => crayon-router-react
crayon-preact => crayon-router-preact
crayon-transition => crayon-router-transition
crayon-svelte => crayon-router-svelte
crayon-kit => crayon-router-kit
crayon-animate => crayon-router-animate

You only need to replace your package.json dependencies and imports to get your builds to work again.

**Comment by alshdavid** [comment]
Oh my god, I thought no one was using this!

I'll republish the packages ASAP - very sorry!
